packages/frameworks/src/frameworks.ts

```diff
@@ -18,22 +18,20 @@ const { readdir, readFile, unlink } = promises;
 
 export const frameworks = [
   {
-    name: 'Blitz.js',
+    name: 'Blitz.js (Legacy)',
```

Comment:
Should there be an entry for non-legacy then?

Reply:
@leerob no, it's just nextjs. Nothing special for blitz

Reply:
Got it 🙏